components/ipfs_ui/components/app.tsx

```diff
@@ -63,6 +63,9 @@ export class IPFSPage extends React.Component<Props, {}> {
     this.actions.getAddressesConfig()
     this.actions.getRepoStats()
     this.actions.getNodeInfo()
+    if (this.props.ipfsData.daemonStatus.launched) {
+      setTimeout(this.refreshActions, 2000)
```

Comment:
I think we should use `setInterval` in `componentDidMount` and call it more than once so the connected peers and other stats always stay up to date while the tab is open.